Client/mods/deathmatch/logic/CElementDeleter.cpp

```diff
@@ -19,14 +19,17 @@ CElementDeleter::CElementDeleter()
     m_bAllowUnreference = true;
 }
 
-void CElementDeleter::Delete(class CClientEntity* pElement)
+void CElementDeleter::Delete(class CClientEntity* pElement, bool silent)
 {
     // Make sure we don't try to delete it twice
     if (pElement && !IsBeingDeleted(pElement))
     {
         // Before we do anything, fire the on-destroy event
-        CLuaArguments Arguments;
-        pElement->CallEvent("onClientElementDestroy", Arguments, true);
+        if (!silent)
+        {
+            CLuaArguments Arguments;
+            pElement->CallEvent("onClientElementDestroy", Arguments, true);
+        }
```

Comment:
We can keep the event trigger to track deleted textures

Reply:
These temporary textures are internal entities and I'm not sure if it's a good idea to expose them by calling `onClientElementDestroy`.

Reply:
It might be good for debug purposes. I think anonymous textures are exposed in the elements tree already.

Reply:
I've allowed onClientElementDestroy events